src/i18n/locale-fr.json

```diff
@@ -143,6 +143,9 @@
         }
     },
     "acl_management": {
+        "buttons": {
```

Comment:
There is an `actions` section below. Move the label there pls

Reply:
Done